src/api/app.ts

```diff
@@ -98,11 +100,10 @@ export function create(
     }));
 
     app.get('/api/workflows/:namespace/:name',
-        async (req, res) => serve(res, () => (forceNamespaceIsolation ? crd.ns(namespace) : crd.ns(req.params.namespace)).workflows.get(req.params.name)));
```

Comment:
Sorry, missed in during first review. Same change is required for `app.get('/api/workflows')`